dispute_gateway.go

```diff
@@ -0,0 +1,137 @@
+package braintree
+
+import (
+	"context"
+	"encoding/xml"
+)
+
+const (
+	EvidenceCategoryDeviceId                                   = "DEVICE_ID"
+	EvidenceCategoryDeviceName                                 = "DEVICE_NAME"
+	EvidenceCategoryPriorDigitalGoodsTransactionArn            = "PRIOR_DIGITAL_GOODS_TRANSACTION_ARN"
+	EvidenceCategoryPriorDigitalGoodsTransactionDateTime       = "PRIOR_DIGITAL_GOODS_TRANSACTION_DATE_TIME"
+	EvidenceCategoryDownloadDateTime                           = "DOWNLOAD_DATE_TIME"
+	EvidenceCategoryGeographicalLocation                       = "GEOGRAPHICAL_LOCATION"
+	EvidenceCategoryLegitPaymentsForSameMerchandise            = "LEGIT_PAYMENTS_FOR_SAME_MERCHANDISE"
+	EvidenceCategoryMerchantWebsiteOrAppAccess                 = "MERCHANT_WEBSITE_OR_APP_ACCESS"
+	EvidenceCategoryPriorNonDisputedTransactionArn             = "PRIOR_NON_DISPUTED_TRANSACTION_ARN"
+	EvidenceCategoryPriorNonDisputedTransactionDateTime        = "PRIOR_NON_DISPUTED_TRANSACTION_DATE_TIME"
+	EvidenceCategoryPriorNonDisputedTransactionEmailAddress    = "PRIOR_NON_DISPUTED_TRANSACTION_EMAIL_ADDRESS"
+	EvidenceCategoryPriorNonDisputedTransactionIpAddress       = "PRIOR_NON_DISPUTED_TRANSACTION_IP_ADDRESS"
+	EvidenceCategoryPriorNonDisputedTransactionPhoneNumber     = "PRIOR_NON_DISPUTED_TRANSACTION_PHONE_NUMBER"
+	EvidenceCategoryPriorNonDisputedTransactionPhysicalAddress = "PRIOR_NON_DISPUTED_TRANSACTION_PHYSICAL_ADDRESS"
+	EvidenceCategoryProfileSetupOrAppAccess                    = "PROFILE_SETUP_OR_APP_ACCESS"
+	EvidenceCategoryProofOfAuthorizedSigner                    = "PROOF_OF_AUTHORIZED_SIGNER"
+	EvidenceCategoryProofOfDeliveryEmpAddress                  = "PROOF_OF_DELIVERY_EMP_ADDRESS"
+	EvidenceCategoryProofOfDelivery                            = "PROOF_OF_DELIVERY"
+	EvidenceCategoryProofOfPossessionOrUsage                   = "PROOF_OF_POSSESSION_OR_USAGE"
+	EvidenceCategoryPurchaserEmailAddress                      = "PURCHASER_EMAIL_ADDRESS"
+	EvidenceCategoryPurchaserIpAddress                         = "PURCHASER_IP_ADDRESS"
+	EvidenceCategoryPurchaserName                              = "PURCHASER_NAME"
+	EvidenceCategoryRecurringTransactionArn                    = "RECURRING_TRANSACTION_ARN"
+	EvidenceCategoryRecurringTransactionDateTime               = "RECURRING_TRANSACTION_DATE_TIME"
+	EvidenceCategorySignedDeliveryForm                         = "SIGNED_DELIVERY_FORM"
+	EvidenceCategorySignedOrderForm                            = "SIGNED_ORDER_FORM"
+	EvidenceCategoryTicketProof                                = "TICKET_PROOF"
+)
+
+type DisputeGateway struct {
+	*Braintree
+}
+
+type FileEvidenceRequest struct {
+	XMLName    xml.Name `xml:"evidence"`
+	DocumentId string   `xml:"document-id"`
+	Category   string   `xml:"category,omitempty"`
+}
+
+type TextEvidenceRequest struct {
```

Comment:
Would it make these structs clearer if they were prefixed with `Dispute`? Looking at the other Braintree SDKs, they don't seem to have it prefixed, but I think it would make it clearer in the godocs for this SDK, since they would be grouped. What do you think?

Reply:
No problem. Will rename the structs.